### PR TITLE
Order model #23

### PR DESCRIPTION
**Pull request by chloe630**
feat: created order model #23

### CODE REVIEW

File: db/models/order.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 'use strict';
-var Sequelize = require('sequelize')
-var db = require('./index.js')
+const Sequelize = require('sequelize')
+const db = require('./index.js')
+
+
+module.exports = db => db.define('order', {
```
Comment (chloe630) [inline db/models/order.js]: 'order ' is to be changed to 'orders'

File: db/models/order.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+  status: {
+    type: Seqelize.ENUM('Pending','Complete'),
```
Comment (gabrielwr) [inline db/models/order.js]: typo!!! Should be 'Sequelize'

**Review by gabrielwr — COMMENTED** [review]
fix comments and then it should be all good!

**Review by gabrielwr — APPROVED** [review]
looks good!

**Pull request merged**